rita_common/src/tunnel_manager/mod.rs

```diff
@@ -324,45 +311,33 @@ impl Neighbor {
     }
 }
 
-impl Message for GetNeighbors {
-    type Result = Result<Vec<Neighbor>, RitaCommonError>;
-}
-impl Handler<GetNeighbors> for TunnelManager {
-    type Result = Result<Vec<Neighbor>, RitaCommonError>;
-
-    fn handle(&mut self, _: GetNeighbors, _: &mut Context<Self>) -> Self::Result {
-        let mut res = Vec::new();
-        for (_, tunnels) in self.tunnels.iter() {
-            for tunnel in tunnels.iter() {
-                res.push(Neighbor::new(
-                    tunnel.neigh_id,
-                    tunnel.iface_name.clone(),
-                    tunnel.ip,
-                    tunnel.speed_limit,
-                ));
-            }
+pub fn tm_get_neighbors() -> Result<Vec<Neighbor>, RitaCommonError> {
```

Comment:
there's no reason for this to return an error now. Previously we had to return an

Reply:
fixed